util.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Some documentation on the purpose of the script and possible commands here would be valuable.